### DISCUSSION TITLE
Is the app a live project?

### DISCUSSION THREAD

**Discussion by BadPixel89** [KonradIT/goprowifihack#233] [General]
Hey, 

Please tell me you are planning to resume updates on this in light of GoPro removing pro tune features from older cams. I have set my Session5 now using your current app, but I don't seem to see shutter speed as an option.

Would also be nice if it could read the status and highlight current settings. Based on what I can see in the repo that should be possible. 

Thanks though, this is more reliable at connecting than the official app.

Dan.

**Comment by KonradIT** [comment]
Hi, which app are you referring to? CamControl is deprecated. You can use other apps such as [this one](https://play.google.com/store/apps/details?id=com.booman.bluetoothremote) or [this one](https://play.google.com/store/apps/details?id=ch.jaromeyer.gopro_ble_remote).
